CONTRIBUTING.md

```diff
@@ -46,11 +46,12 @@ Run all the unit tests and generate coverage report of the unit tests for sonar.
 
 ### test:e2e
 
-To run all the e2e tests, you need to run the application with terrator-plugin and use specific url for library of template:
+To run all the e2e tests, you need to run the application with ALL official plugins and use specific url for library of template:
 
 ```bash
 npm ci
-npm run plugin:install -- repository-name="terrator-plugin" repository-url="https://github.com/ditrit/terrator-plugin.git#0.3.0"
+npm run plugin:install -- repository-name="terrator-plugin" repository-url="https://github.com/ditrit/terrator-plugin.git#0.3.1"
+npm run plugin:install -- repository-name="githubator-plugin" repository-url="https://github.com/ditrit/githubator-plugin.git#0.2.1"
```

Comment:
reminder to update to the last version of each plugin

Reply:
done